plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/equinox/internal/security/ui/storage/ChangePasswordWizardDialog.java

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.reddeer.eclipse.equinox.internal.security.ui.storage;
+
+import org.jboss.reddeer.common.wait.WaitUntil;
+import org.jboss.reddeer.jface.wizard.WizardDialog;
+import org.jboss.reddeer.swt.condition.ShellIsAvailable;
+import org.jboss.reddeer.swt.impl.button.NoButton;
+import org.jboss.reddeer.swt.impl.button.OkButton;
+import org.jboss.reddeer.swt.impl.button.YesButton;
+import org.jboss.reddeer.swt.impl.group.DefaultGroup;
+import org.jboss.reddeer.swt.impl.text.DefaultText;
+
+/**
+ * Secure storage change master password dialog.
+ *
+ * @author jnovak@redhat.com
+ */
+public class ChangePasswordWizardDialog extends WizardDialog {
+
+	private static final int HINTS_COUNT = 2;
+
+	public ChangePasswordWizardDialog() {
+		super("Change Password");
+	}
+
+	public void ignoreHints() {
+		new WaitUntil(new ShellIsAvailable("Secure Storage - Password Hint Needed"));
```

Comment:
This is not a good approach since the shell might not be activated. Just use 
```
new DefaultShell("...")
```
Note that the above code waits for TimePeriod.DEFAULT for the shell.